gnucash/gnucash-cli.cpp

```diff
@@ -114,10 +114,10 @@ Gnucash::GnucashCli::configure_program_options (void)
 
     bpo::options_description cli_options(_("Scripting and/or Interactive Session Options"));
     cli_options.add_options()
-        ("script,S", bpo::value (&m_script), "Script to run")
-        ("interactive,I", bpo::bool_switch (&m_interactive), "Interactive session")
-        ("language,L", bpo::value (&m_language)->default_value("guile"), "Specify language for script or interactive session; guile (default) or python")
-        ("readwrite,W", bpo::bool_switch (&m_open_readwrite), "Open datafile read-write for script and/or interactive session");
+        ("script,S", bpo::value (&m_script), _("Script to run"))
+        ("interactive,I", bpo::bool_switch (&m_interactive), _("Interactive session"))
+        ("language,L", bpo::value (&m_language)->default_value("guile"), _("Specify language for script or interactive session; guile (default) or python"))
+        ("readwrite,W", bpo::bool_switch (&m_open_readwrite), _("Open datafile read-write for script and/or interactive session"));
```

Comment:
The commit message says 'some stl algos' :wink:

Reply:
It's a temporary commit which will be merged up later on😭

Reply:
Ooh, my apologies for not getting that...